bandit.yaml

```diff
@@ -0,0 +1,109 @@
+# optional: after how many files to update progress
+#show_progress_every: 100
+
+# optional: plugins directory name
+#plugins_dir: 'plugins'
+
+# optional: plugins discovery name pattern
+plugin_name_pattern: '*.py'
+
+# optional: terminal escape sequences to display colors
+#output_colors:
+#    DEFAULT: '\033[0m'
+#    HEADER: '\033[95m'
+#    INFO: '\033[94m'
+#    WARN: '\033[93m'
+#    ERROR: '\033[91m'
+
+# optional: log format string
+#log_format: "[%(module)s]\t%(levelname)s\t%(message)s"
+
+# globs of files which should be analyzed
+include:
+    - '*.py'
+    - '*.pyw'
+
+# a list of strings, which if found in the path will cause files to be excluded
+# for example /tests/ - to remove all all files in tests directory
+exclude_dirs:
+    - '/tests/'
```

Comment:
Excluding `tests/`, we pass all the checks. Including `tests/`, we get dinged on the use of `eval` in a few tests that test `repr` and for using `TLSv1` in one client test. The TLS one is easy to fix. The `eval` ones technically are too, but I'd like to keep those tests as they are, since they're doing exactly what I want them to.